src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -160,6 +160,9 @@ internal enum MessageID
         IDS_FeatureExpressionVariablesInQueriesAndInitializers = MessageBase + 12742,
         IDS_FeatureExtensibleFixedStatement = MessageBase + 12743,
         IDS_FeatureIndexingMovableFixedBuffers = MessageBase + 12744,
+
+        IDS_FeatureIndexOperator = MessageBase + 12745,
```

Comment:
>IDS_FeatureIndexOperator [](start = 8, length = 24)

Do we have to have two separate features?

Reply:
I prefer that for clarity reasons. I don't want to say "you cannot do this because of "indexes and ranges" feature"

---
In reply to: [185362807](https://github.com/dotnet/roslyn/pull/26405#discussion_r185362807) [](ancestors = 185362807)